gulpfile.js

```diff
@@ -0,0 +1,39 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var rename = require('gulp-rename');
+var cleanCSS = require('gulp-clean-css');
+var autoprefixer = require('gulp-autoprefixer');
+var browserSync = require('browser-sync').create();
+
+gulp.task('default', ['serve']);
+
+gulp.task('serve', ['minify-css'], function() {
+  browserSync.init({
+    server: {
+      baseDir: "./"
+    }
+  });
+  
+  gulp.watch("./src/**/*.sass", ['sass']);
+});
+
+gulp.task('sass', function() {
+  return gulp.src("./src/legacy.sass")
+  .pipe(sass())
```

Comment:
Hmm. This is more a personal style convention than general js style convention. Gulpfiles typically use one standard indentation past the leftmost return statement. I would prefer to stay consistent with gulp indentation; the change will be reflected in the next commit.